DRODLib/CharacterCommand.h

```diff
@@ -324,6 +324,7 @@ class CCharacterCommand
 		CC_PushTile,            //Push object or entity at (x,y) in direction (w)
 		CC_SetMovementType,     //Sets the character's movement type to X.
 		CC_ReplaceWithDefault,  //Replaces the character's script with its default script (if possible)
+		CC_VarSetAt,            //Remotely set local variable w (with operation h) of NPC at (x,y)
```

Comment:
Let's add one more detail here, including how flags is used as a relevant parameter, e.g.,
`//Remotely set local variable w (with operation h) of NPC at (x,y) to value in flags`